services/paymaster/src/main.rs

```diff
@@ -1,43 +1,64 @@
-use crate::config::load_config;
+use std::sync::Arc;
+
+extern crate dotenv;
 use clap::Parser;
+use dotenv::dotenv;
 use opentelemetry::trace::TracerProvider;
 use opentelemetry_otlp::WithExportConfig;
+use tokio::sync::RwLock;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
 mod api;
-mod config;
+mod config_manager;
 mod constraint;
 mod constraint_templates;
+mod db;
 mod metrics;
 mod rpc;
 mod serde;
 
-#[derive(Parser)]
+type DomainStateMap = std::collections::HashMap<String, api::DomainState>;
+type SharedDomains = Arc<RwLock<DomainStateMap>>;
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about, long_about = None)]
```

Comment:
I feel like `author` and `long_about` can be removed since the crate has no `author` in the manifest and `long_about = None` is a noop in this context